src/deploy/database/prepare.ts

```diff
@@ -31,7 +30,7 @@ module.exports = function (context, options) {
     deploys.push(ruleConfig);
   });
 
-  _.forEach(ruleFiles, function (v, file) {
+  _.forEach(ruleFiles, (v, file) => {
```

Comment:
Prefer for loop over Object.entries

Reply:
ack - will do a pass for more of these in the near future